install-and-configure/advanced-configuration/user-management-oidc/microsoft-entra-id-oidc-integration-for-kubecost.md

```diff
@@ -120,3 +120,5 @@ kubecostModel:
     - name: LOG_LEVEL
       value: debug
 ```
+
+### Option 4: Make sure there are no optional claims in Enrata App Manage-->Token configuration page
```

Comment:
I think this section could use a title, then one or two sentences providing clear details on where the "Optional Claims" are in the EntraID portal.

Also, be careful here! Looks like you mispelled Entra with "Enrata".

Reply:
thank you!